carbon-location.html

```diff
@@ -148,8 +148,8 @@
         }
       },
 
-      __computeRoutePath: function(path, hash, useHashAsPath) {
-        return useHashAsPath ? hash : path;
+      __computeRoutePath: function() {
+        return this.useHashAsPath ? this.__hash : this.__path;
```

Comment:
Given the nature of this change, I feel I must ask: is this a stylistic change, or does it actually affect the re-entrancy conditions?